website/docs/reference/datasources/querying-oracle.md

```diff
@@ -1,219 +1,101 @@
 ---
 sidebar_position: 11
+description: Connect Appsmith to an Oracle database and create queries.
 ---
 
 # Oracle
 
-This page describes how to connect your application to your Oracle databases and use queries to manage its content.
+This page gives information to connect Appsmith to an Oracle database and to query its data in your applications.
 
-## Configuration
+## Connection parameters
+
+The following is a reference guide that provides a description of the parameters for connecting to an Oracle database.
 
 <figure>
   <img src="/img/oracle-datasource-config.png" style={{width: "100%", height: "auto"}} alt="Configuring an Oracle datasource." />
   <figcaption align="center"><i>Configuring an Oracle datasource.</i></figcaption>
 </figure>
 
-:::tip
-If you want to connect to a locally hosted database, you can use a service like ngrok to expose it. For more information, see [How to connect to local database on Appsmith](/advanced-concepts/more/how-to-work-with-local-apis-on-appsmith).
+:::caution important
+If you are a self-hosted user, you must whitelist the IP addresses 18.223.74.85 and 3.131.104.27 of the Appsmith deployment on your database instance. See [Managing Internet Protocol Allowlist and Blocklist Rules](https://docs.oracle.com/en/cloud/get-started/subscriptions-cloud/mmocs/managing-internet-protocol-whitelist-and-blacklist-rules.html) for more details.
 :::
 
-To connect to your database, Appsmith needs the following parameters:
+<dl>
+  <dt><b>Host Address</b></dt>
+  <dd>The network location of your Oracle database. This can be a domain name or an IP address. To connect to a local Oracle database, see <a href="/advanced-concepts/more/how-to-work-with-local-apis-on-appsmith"><b>Connect Local Database</b></a> for directions. </dd><br />
 
-* **Host Address:** Provide the hostname or IP address of your database server. If you are connecting to a self-hosted database, see [this guide](/advanced-concepts/more/how-to-work-with-local-apis-on-appsmith).
+  <dt><b>Port</b></dt>
+  <dd>The port number to connect to on the server. </dd><br />
 
-* **Port:**  Provide the port on which to connect to your database.
+  <dt><b>Service Name</b></dt>
+  <dd>The service name for your database instance. </dd><br />
 
-* **Service Name:** Provide the service name for your database instance.
+  <dt><b>Authentication</b></dt>
+  <dd>The <b>Username</b> and <b>Password</b> for the user with which you are connecting to the database.</dd><br />
 
-* **Authentication:** Provide the **Username** and **Password** for the user with which you are connecting to the database.
+  <dt><b>SSL Mode</b></dt>
+  <dd>Determines whether your queries use an SSL connection to communicate with the database.</dd><br />
+  <dd><i>Options:</i>
+    <ul>
+      <li><b>TLS:</b> Appsmith attempts to connect using TLS.</li>
```

Comment:
these look incomplete. Can we use the one given in the draft. When we say attempt in TLS, it could mean that the encryption is either required or allowed. There shouldn't be any ambiguity here.